applications/RomApplication/custom_strategies/rom_builder_and_solver.h

```diff
@@ -96,15 +98,18 @@ class ROMBuilderAndSolver : public BuilderAndSolver<TSparseSpace, TDenseSpace, T
     typedef typename BaseType::ConditionsArrayType ConditionsArrayType;
 
     /// Additional definitions
+    typedef typename ModelPart::MasterSlaveConstraintContainerType MasterSlaveConstraintContainerType;
     typedef Element::EquationIdVectorType EquationIdVectorType;
     typedef Element::DofsVectorType DofsVectorType;
     typedef boost::numeric::ublas::compressed_matrix<double> CompressedMatrixType;
+    typedef LocalSystemMatrixType RomSystemMatrixType;
+    typedef LocalSystemVectorType RomSystemVectorType;

```

Comment:
I would use the original names for these as these are the ones we are used to (used in the majority of Kratos schemes and B&Ss).

Reply:
I wanted to be wanted explicit about the distinction between the large system matrices(Ax = b) and the much smaller reduced matrices Arom*q = brom.

These will have different design constraints because they are much smaller and don't need to be distributed when MPI is implemented, so I gave them their own types just to future-proof them.